cxx4opencl/address_spaces.txt

```diff
@@ -420,3 +420,27 @@ gengenptr = addrspace_cast<int**>(cnstgenptr); // illegal.
 gengenptr = reinterpret_cast<int**>(locgenptr); // legal.
 gengenptr = reinterpret_cast<int**>(cnstgenptr); // legal.
 ----------
+
+[[remove-addrspace]]
+==== Address space removal
+
+Utility `+__remove address_space+` allows to derive types that have address
```

Comment:
Since we haven't supported this in C++ for OpenCL 1.0 initially, I suggest we add this only in 2021 version? so maybe we can add some text to clarify that and then also dependency on https://github.com/KhronosGroup/OpenCL-Docs/pull/616?